programs/perpetuals/tests/anchor/basic.ts

```diff
@@ -20,6 +25,7 @@ describe("perpetuals", () => {
   let multisigExpected;
   let tokenExpected;
   let positionExpected;
+  let permissionlessOracleUpdateKey;
```

Comment:
also lets rename it here and move to test_client.ts where all similar keys are generated

Reply:
done!